lib/web/ui/app.go

```diff
@@ -155,8 +155,14 @@ func MakeApp(app types.Application, c MakeAppsConfig) App {
 
 // MakeAppTypeFromSAMLApp creates App type from SAMLIdPServiceProvider type for the WebUI.
 // Keep in sync with lib/teleterm/apiserver/handler/handler_apps.go.
+// Note: The SAMLAppPreset field is used in SAML service provider update flow in the
+// Web UI. Thus, this field is currently not available in the Connect App type.
 func MakeAppTypeFromSAMLApp(app types.SAMLIdPServiceProvider, c MakeAppsConfig) App {
 	labels := makeLabels(app.GetAllLabels())
+	samlAppPreset := "unspecified"
+	if app.GetPreset() != "" {
+		samlAppPreset = app.GetPreset()
+	}
```

Comment:
```suggestion
	samlAppPreset := cmp.Or(app.GetPreset(), "unspecified")
```

Reply:
Wow, didn't knew this was possible. I never used the cmp package besides the Diff func in tests.

https://github.com/gravitational/teleport/pull/43863/commits/b60eb520684adb661c850f47d21691a22739e3c3

Reply:
That's go-cmp, not cmp from the standard library.